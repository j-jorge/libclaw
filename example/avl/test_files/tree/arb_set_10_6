 93 73 8 40 0 34 74 70 83 86 80