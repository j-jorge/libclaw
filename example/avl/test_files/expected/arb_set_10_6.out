Tree is empty
File loading...
size : 11
Min = 0
Max = 93
Tree's not empty
Found : 0 8 34 40 70 73 74 80 83 86 93
Scan with an iterator : 0 8 34 40 70 73 74 80 83 86 93
Scan bis, iterator affectation :0: 0 8 34 40 70 73 74 80 83 86 93
8: 8 34 40 70 73 74 80 83 86 93
34: 34 40 70 73 74 80 83 86 93
40: 40 70 73 74 80 83 86 93
70: 70 73 74 80 83 86 93
73: 73 74 80 83 86 93
74: 74 80 83 86 93
80: 80 83 86 93
83: 83 86 93
86: 86 93
93: 93
Deletions
Copy and scan
source size : 0
copy size : 0
Tree's copy is empty

Reverse scan

Assignment
Clear
Tree's is empty
